model/textparse/promparse_test.go

```diff
@@ -492,7 +492,7 @@ const (
 	promtestdataSampleCount = 410
 )
 
-func BenchmarkParse(b *testing.B) {
+func BenchmarkPromParse(b *testing.B) {
```

Comment:
Hm, why this change? Are we not benchmark the openmetrics case as well here? 🤔

Reply:
We are but made this distinction because there's a `BenchmarkParse` in openmetrics test too

Rational here is  Prom and OM here are to indicate the input text

Reply:
Got it, thanks! It looks like the benchmark code is generally duplicated though 🤔